### DISCUSSION TITLE
See exchange through yfinance?

### DISCUSSION THREAD

**Discussion by rhwvu** [ranaroussi/yfinance#2037] [Q&A] [answered]
Simple question, is there a way to see the exchange a stock is traded on through yfinance? With MSFT as an example, it's included at the top of most pages with "NasdaqGS - Nasdaq Real Time Price", where "NasdaqGS" would theoretically be the thing I'd want to grab. As far as I can tell, though, it's not included in anything yfinance usually scrapes. Anyone have any advice?

**Comment by ValueRaider** [comment, marked as answer]
history metadata

**Reply by rhwvu** [reply]
Thank you, that's very helpful for my little project.
